app/src/main/java/com/codesoom/assignment/service/TaskService.java

```diff
@@ -0,0 +1,26 @@
+package com.codesoom.assignment.service;
+
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+
+import com.codesoom.assignment.model.Task;
+
+@Service
+public interface TaskService {
+
+
+    List<Task> getTaskListService();
+
+
+    Task createTaskService(Task task);
+
+
+    Task getTaskService(Long id);
+
+
+    Task modifyTaskService(Long id, String title);
+
+
+    void deleteTaskService(Task task);
```

Comment:
메소드 이름에 모두 `Service`가 들어간 것이 어색합니다.
```suggestion
    List<Task> getTaskList();

    Task createTask(Task task);

    Task getTask(Long id);

    Task modifyTask(Long id, String title);


    void deleteTask(Task task);
```

Reply:
이름 짓기가 어려워서 그냥 컨트롤러에 있는 메서드에서 끝에 Service만 붙였는데 어떻게 하면 적절한 이름을 지어줄 수 있을까요?

Reply:
수정하였습니다.

Reply:
먼저 메소드 이름에서 `Service`를 빼시면 됩니다. 그 다음에 더 좋은 이름을 생각해보시죠.